src/helix/endpoints/chat/get_user_emotes.rs

```diff
@@ -0,0 +1,184 @@
+    #[cfg_attr(feature = "typed-builder", builder(default))]
+    #[cfg_attr(feature = "deser_borrow", serde(borrow = "'a"))]
+    pub after: Option<Cow<'a, helix::CursorRef>>,
+    /// The User ID of a broadcaster you wish to get follower emotes of. Using this query parameter will guarantee inclusion of the broadcaster’s follower emotes in the response body.
+    #[cfg_attr(feature = "typed-builder", builder(setter(into)))]
+    #[cfg_attr(feature = "deser_borrow", serde(borrow = "'a"))]
+    pub broadcaster_id: Option<Cow<'a, types::UserIdRef>>,
+}
+
+impl<'a> GetUserEmotesRequest<'a> {
+    /// Get emotes in these sets
+    pub fn user_id(user_id: impl types::IntoCow<'a, types::UserIdRef> + 'a) -> Self {
+        Self {
+            user_id: user_id.into_cow(),
+            after: None,
+            broadcaster_id: None,
+        }
+    }
+}
+
+impl helix::Paginated for GetUserEmotesRequest<'_> {
+    fn set_pagination(&mut self, cursor: Option<helix::Cursor>) {
+        self.after = cursor.map(|c| c.into_cow())
+    }
+}
+
+/// Return Values for [Get User Emotes](super::get_user_emotes)
+///
+/// [`get-user-emotes`](https://dev.twitch.tv/docs/api/reference/#get-user-emotes)
+#[derive(PartialEq, Eq, Deserialize, Serialize, Debug, Clone)]
+#[cfg_attr(feature = "deny_unknown_fields", serde(deny_unknown_fields))]
+#[non_exhaustive]
+pub struct UserEmote {
+    /// Emote ID.
+    pub id: types::EmoteId,
+    /// Name of the emote a viewer types into Twitch chat for the image to appear.
+    pub name: String,
+    // FIXME: Enumify?
+    /// The type of emote.
+    pub emote_type: String,
+    /// ID of the emote set the emote belongs to.
+    pub emote_set_id: types::EmoteSetId,
+    /// User ID of the broadcaster who owns the emote.
+    pub owner_id: types::UserId,
+    /// The formats that the emote is available in.
+    pub format: Vec<types::EmoteAnimationSetting>,
+    /// The sizes that the emote is available in.
+    pub scale: Vec<types::EmoteScale>,
+    /// The background themes that the emote is available in.
+    pub theme_mode: Vec<types::EmoteThemeMode>,
+}
+
+impl UserEmote {
+    /// Create an emote builder for this emote.
+    ///
+    /// # Examples
+    ///
+    /// ```rust, no_run
+    /// use twitch_api::helix::{self, chat::get_channel_emotes};
+    /// use futures::TryStreamExt;
+    /// # use twitch_api::{client, types};
+    /// # #[tokio::main]
+    /// # async fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync + 'static>> {
+    /// # let client: helix::HelixClient<'static, client::DummyHttpClient> = helix::HelixClient::default();
+    /// # let token = twitch_oauth2::AccessToken::new("validtoken".to_string());
+    /// # let token = twitch_oauth2::UserToken::from_existing(&client, token, None, None).await?;
+    /// let emotes: Vec<_> = client.get_user_emotes("301590448", &token).try_collect().await?;
```

Comment:
```suggestion
    /// let emotes: Vec<_> = client.get_user_emotes(&token).try_collect().await?;
```

This doesn't take a user-id - maybe there should be something that takes one? The only use-case I see is for custom tokens that don't have the user-id set.

Reply:
I think if they have such a custom token they can be bothered with using the more manual route :3 a twitch_oauth2 `TwitchToken` will always have a user_id if `/validate` says there is one